### ISSUE TITLE
Labels format to Finetune SuperPoint

### ISSUE THREAD

**Issue by jack-turkey** [rpautrat/SuperPoint#190]
Thank you for uploading the material and the instructions to fine-tune SuperPoint. The  problem I am facing in the finetuning of the SuperPoint model is that from your material I am not able to understand how the keypoints are saved/stored in each label file, in order to replace them with my own labels. It would be really really helpful for me if you help me with this, since otherwise i am not able to proceed.

Thank you very much in advance!

**Comment by rpautrat** [comment]
Hi!
The ground truth keypoints are saved [here](https://github.com/rpautrat/SuperPoint/blob/6d830927e89308d353b5b368a76acb3405d5bada/superpoint/export_detections.py#L74) in `superpoint/export_detections.py`. They are stored as one npz file per image, and the key 'points' give you access to the exported point coordinates (Nx2 array, in ij convention if I remember correctly).

**Comment by rpautrat** [comment]
Did it answer your question? If yes, feel free to close the issue!
